src/Qwiq.Core/IQueryFolder.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Microsoft.Qwiq
+{
+    public interface IQueryFolder : IIdentifiable<Guid>, IEquatable<IQueryFolder>
+    {
+        string Name { get; }
+        IQueryFolderCollection SubFolders { get; }
```

Comment:
This name doesn't make sense to me--the folder really is a folder hierarchy, and in a hierarchy, we have `IQueryFolderCollection` containing children, so I'd expect this to be more like the `INode` interface

Reply:
I'll take a look and update appropriately